example/CMakeLists.txt

```diff
@@ -23,7 +23,13 @@ if (BUILD_READERS)
 endif()
 
 if (BUILD_READERS)
-    find_package(Qt5 COMPONENTS Gui Multimedia Quick)
+    find_package(QT NAMES Qt5 Qt6 REQUIRED COMPONENTS Gui Multimedia Quick)
```

Comment:
I assume that the `REQUIRED` clause breaks the build if there is no qt: https://github.com/zxing-cpp/zxing-cpp/actions/runs/3585591723/jobs/6040040429.

I'd also prefer not to mix the message cosmetic with a change of the used Qt version in one commit.